packages/vscode-extension/src/webview/views/LaunchConfigurationView.tsx

```diff
@@ -614,7 +614,7 @@ function EasBuildConfiguration({
           name={`eas.${platform}.profile`}>
           {availableEasBuildProfiles.map((profile) => (
             <Option key={profile.value} value={profile.value} disabled={profile.disabled}>
-              {profile.label}
+              {profile.label} {profile.disabled ? "(not a simulator profile)" : ""}
```

Comment:
Should we show this for Android as well? Maybe the message should be tweaked to match the platform?